### ISSUE TITLE
Support setting the dashboard key as an environment variable

### ISSUE THREAD

**Issue by jszwedko** [deployd/deployd#725]
When deploying to environments like Heroku you cannot depend on the local filesystem being persistent (you cannot actually open a shell on the same servers running the web server processes). Thus it would be useful to tell deployd what key to use via environment variables.

Would you be open to a PR adding this functionality?

**Comment by rgolea** [comment]
@jszwedko yes, we would be open for a PR... Maybe @NicolasRitouet can also confirm. Although... for security reasons you might want to just disable the dashboard and the dpd.js frontend library. But this is just my point of view.

**Comment by jszwedko** [comment]
Awesome! I'll see if I can find some time to put something together.

I agree in principle about disabling it, but the dashboard is useful as a makeshift "admin" until we can throw together an admin front-end.

**Comment by rgolea** [comment]
I would strongly recommend you would use this app instead of the deployd dashboard for maintainance:
http://mongotron.io/#/

**Comment by jszwedko** [comment]
Nifty! I am unfamiliar with this tool -- using deployd is my first foray into MongoDB -- but this seems like it would fit the bill.
